recipes/setup_raid_on_head_node.rb

```diff
@@ -59,6 +59,7 @@
   # Create RAID device with mdadm
   raid_superblock_version = value_for_platform(
     'centos' => { '>=8' => '1.2' },
+    'ubuntu' => { '>=20.04' => '1.2' },
```

Comment:
what is happening for U18 case? is it going to use the default value?

Reply:
Yes, 0.90 is used on Ubuntu 18